docs/content/doc/advanced/config-cheat-sheet.en-us.md

```diff
@@ -443,7 +443,7 @@ relation to port exhaustion.
     - disabled - show an error
     - login - show an account linking login
     - auto - link directly with the account, please be aware that it may automatically merge two accounts
-      only because they have same names.
+      only because they have same names/emails depending on the setting you choose to defariate nickname & identity from.
```

Comment:
Is "defariate" a typo or am I missing something?

Reply:
typo I would say ... I'm not good at wordings ...

Reply:
Unfortunatell same for me. It will be good if someone will be able to correct this. Also it will be good to finish with the ouath2 auto register PR and to move forward...